ui/tests/acceptance/enterprise-replication-test.js

```diff
@@ -235,16 +234,8 @@ module('Acceptance | Enterprise | replication', function(hooks) {
 
     // checks on secondary token modal
     assert.dom('#modal-wormhole').exists();
-    const today = new Date();
-    today.setMinutes(today.getMinutes() + 30); // add default 30 min TTL to current date to return expires
-    const dateFormatted = dateFormat([today, 'MMM DD, YYYY hh:mm:ss A']);
-    const modalDate = document.querySelector('[data-test-row-value="Expires"]').innerText;
-    // because timestamp might be off by a 1 sec due to different in exp from token vs. adding 30 min, checking the string before the seconds.
-    assert.equal(
-      dateFormatted.slice(0, 18),
-      modalDate.slice(0, 18),
-      'shows the correct expiration date and in the correct format'
-    );
+    const modalDefaultTtl = document.querySelector('[data-test-row-value="TTL"]').innerText;
```

Comment:
Should we do one more test scenario where we don't use the default (set it to something specific) and make sure the TTL reflects that?

Reply:
Good call.  Just pushed a commit that checks for when it's set at 3 minutes as well.